source/blog/2016-04-20-scala-210-and-java7.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: Final Scala 2.10 and Java 7 releases
+published: true
+post_author:
+  display_name: Kevin Oliver
+  twitter: kevino
+tags: Finagle, Finatra, Util, Scrooge, TwitterServer, Ostrich
+---
+
+The upcoming releases of our family of projects will be the final release for
+Scala 2.10 and Java 7. These should be:
+
+* Finagle 6.35.0
+* Finatra 2.2.0
```

Comment:
I think our  2.11 only release would be the 2.2.0 release as we'd keep continuity in our 2.1.x releases being Scala 2.10 compatible. But maybe that doesn't make sense?